vcd/resource_vcd_independent_disk.go

```diff
@@ -173,33 +197,180 @@ func resourceVcdIndependentDiskCreate(d *schema.ResourceData, meta interface{})
 
 	diskCreateParams.Disk.Description = d.Get("description").(string)
 
+	if value, ok := d.GetOk("sharing_type"); ok {
```

Comment:
Should we immediately validate at runtime if this feature is supported (API version is sufficient) and return an error

Reply:
added